code/modules/surgery/organs/subtypes/skrell.dm

```diff
@@ -1,3 +1,51 @@
+#define HEADPOCKET_DAMAGE_THRESHOLD 8
+
 /obj/item/organ/internal/liver/skrell
 	alcohol_intensity = 4
 	species = "Skrell"
+
+/obj/item/organ/internal/headpocket
+	name = "headpocket"
+	desc = "Allows Skrell to hide tiny objects within their head tentacles."
+	icon_state = "skrell_headpocket"
+	origin_tech = "biotech=2"
+	w_class = WEIGHT_CLASS_TINY
+	parent_organ = "head"
+	slot = "headpocket"
+	species = "Skrell"
+	actions_types = list(/datum/action/item_action/organ_action/toggle)
+	var/obj/item/weapon/storage/internal/pocket
+	var/head_damage = 0
+
+/obj/item/organ/internal/headpocket/New()
+	..()
+	pocket = new /obj/item/weapon/storage/internal(src)
+	pocket.storage_slots = 1
+	// Allow adjacency calculation to work properly
+	loc = owner
+	// Fit only pocket sized items
+	pocket.max_w_class = 2
+	pocket.max_combined_w_class = 2
+
+/obj/item/organ/internal/headpocket/on_life()
+	..()
+	var/obj/item/organ/external/head/head = owner.get_organ("head")
+	if(pocket.contents.len &&	(head.brute_dam - head_damage >= HEADPOCKET_DAMAGE_THRESHOLD ||	!findtextEx(head.h_style, "Tentacles")))
+		owner.visible_message("<span class='notice'>Something falls from [owner]'s head!</span>",
+													"<span class='notice'>Something falls from your head!</span>")
+		pocket.empty_object_contents(0, get_turf(owner))
+	head_damage = head.brute_dam
+
+/obj/item/organ/internal/headpocket/ui_action_click()
+	if(!loc)
+		loc = owner
```

Comment:
Hopefully this doesn't break anything? Internal organs are allowed to have a loc right? If not, I can just set loc back to null after the MouseDrop call.